flask_mqtt/__init__.py

```diff
@@ -109,6 +109,14 @@ def init_app(self, app: Flask, config_prefix : str = "MQTT") -> None:
         
         if self.app is None:
             self.app = app
+
+        if 'mqtt' in app.extensions:
+            raise RuntimeError(
```

Comment:
This is open to discussion, since it could break existing projects if devs already define their own 'mqtt' extension.